chatterbot/ext/django_chatterbot/models.py

```diff
@@ -2,16 +2,34 @@
 
 
 class Statement(models.Model):
+    """A short (<255) chat message, tweet, forum post, etc"""
 
     text = models.CharField(
         unique=True,
         blank=False,
         null=False,
+        default='<empty>',
```

Comment:
I'm not sure about having a default response specified because it will alter how the chat bot handles knowledge it has accumulated for conversations. If a default of `'<empty>'` is set, the chat bot will then be able to respond to a user's input with a response of '<empty>'.

Was there a reason for adding this?